packages/autocomplete-plugin-redirect-url/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@algolia/autocomplete-plugin-redirect-url",
+  "description": "A plugin to add out-of-the-box redirect URL functionality to Algolia Autocomplete.",
```

```suggestion
  "description": "A plugin to add redirect URL functionality to Algolia Autocomplete.",
```

we probably want to get rid of ootb and out-of-the-box everywhere